src/components/Database/DataViewer/InlineQuery.tsx

```diff
@@ -14,26 +14,27 @@
  * limitations under the License.
  */
 
-import * as React from 'react';
-import {
-  QueryParams,
-  DEFAULT_PAGE_SIZE,
-  DEFAULT_QUERY_PARAMS,
-  jsonIshValue,
-} from './common/view_model';
-import { Select } from '@rmwc/select';
-import { TextField } from '@rmwc/textfield';
-import { useState, useEffect, useRef } from 'react';
-import keycode from 'keycode';
+import './InlineQuery.scss';
```

Comment:
DId this get auto-moved to the top?

Reply:
yep, side effect imports (no variable assignment) are put at top.